pvt/helpers/src/models/vault/VaultDeployer.ts

```diff
@@ -42,14 +68,30 @@ export default {
     return deploy('v2-pool-utils/MockVault', { from, args: [authorizer.address] });
   },
 
-  async _deployAuthorizer(admin: SignerWithAddress, from?: SignerWithAddress): Promise<Contract> {
-    return deploy('v2-vault/TimelockAuthorizer', { args: [admin.address, ZERO_ADDRESS, MONTH], from });
+  async _deployAuthorizer(
+    admin: SignerWithAddress,
+    authorizerAdaptorEntrypoint: Contract,
+    from?: SignerWithAddress,
+    vault?: string
+  ): Promise<Contract> {
+    return deploy('v2-vault/TimelockAuthorizer', {
+      args: [admin.address, vault || ZERO_ADDRESS, authorizerAdaptorEntrypoint.address ?? ZERO_ADDRESS, MONTH],
```

Comment:
the entrypoint gets the adaptor in the `TimelockAuthorizer` constructor; doesn't this revert if we send it the `ZERO_ADDRESS`? Why would the address be undefined if the entrypoint is a `Contract`?

Reply:
Correct - this is leftover from when it was optional. It may change again, but yes, I can remove that.